athena/modules/text/module_text_llm/module_text_llm/cot_learner_profile/generate_suggestions.py

```
@@ -4,115 +4,138 @@
-async def generate_suggestions(exercise: Exercise, submission: Submission, config: ApproachConfig, *, debug: bool,
-                               is_graded: bool, learner_profile: Optional[LearnerProfile] = None) -> List[Feedback]:
+async def generate_suggestions(
+    exercise: Exercise,
+    submission: Submission,
+    config: COTLearnerProfileConfig,
+    *,
+    debug: bool,
+    is_graded: bool,
+    learner_profile: Optional[LearnerProfile] = None,
+) -> List[Feedback]:
     model = config.model.get_model()  # type: ignore[attr-defined]
```

Ruff (0.11.9)

Local variable `model` is assigned to but never used

Remove assignment to unused variable `model`

(F841)